taskcluster/scripts/build/windows.ps1

```diff
@@ -53,16 +53,18 @@ $ErrorActionPreference = "Stop"
 mkdir $TASK_WORKDIR/cmake_build 
 $BUILD_DIR =resolve-path "$TASK_WORKDIR/cmake_build"
 
-cmake -S . -B $BUILD_DIR -GNinja
-cmake --build $BUILD_DIR --config RelWithDebInfo
-cmake --build $BUILD_DIR --config RelWithDebInfo --target msi
+
+cmake --version
+cmake -S . -B $BUILD_DIR -GNinja -DCMAKE_BUILD_TYPE=Release # TODO: Linking breaks horribly with RelWithDebInfo
+cmake --build $BUILD_DIR #--config RelWithDebInfo Ignored as we are using ninja
+cmake --build $BUILD_DIR --target msi
 cmake --install $BUILD_DIR --prefix "$TASK_WORKDIR/unsigned"
 
 Write-Output "Writing Artifacts"
 New-Item -ItemType Directory -Path "$TASK_WORKDIR/artifacts" -Force
 $ARTIFACTS_PATH =resolve-path "$TASK_WORKDIR/artifacts"
 Copy-Item -Path $BUILD_DIR/windows/installer/MozillaVPN.msi -Destination $ARTIFACTS_PATH/MozillaVPN.msi
-Copy-Item -Path "$TASK_WORKDIR/unsigned/Mozilla VPN.pdb" -Destination "$ARTIFACTS_PATH/Mozilla VPN.pdb"
```

Comment:
Could you explain what's going on with this change and where `vc140.pdb` comes from?

Reply:
Building with `/ZI` will force the generation of the .pdb and vc140 is just the [default name](https://docs.microsoft.com/en-us/cpp/build/reference/fd-program-database-file-name?view=msvc-170) - tried to add flags to re-name it, but that failed so i did not investigate further.